src/Resources/views/CRUD/base_array_macro.html.twig

```diff
@@ -8,17 +8,42 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% macro render_array(value, inline) %}
-    {% from _self import render_array %}
-    {% for key, val in value %}
-        {% if val is iterable %}
-            [{{ key }} => {{ render_array(val, inline) }}]
-        {% else %}
-            [{{ key }} => {{ val }}]
-        {% endif %}
+{% macro render_array(value, inline, values_only, translate, translation_domain, translation_prefix, translate_key) %}
+    {% spaceless %}
+        {% if not inline %}<ul>{% endif %}
+        {% from _self import render_array %}
+        {% for key, val in value %}
+            {% if inline == true %}
+                [
+            {% else %}
+                {% if val is not iterable or not values_only %}
+                    <li>
+                {% endif %}
+            {% endif %}
 
-        {% if not loop.last and not inline %}
-            <br>
-        {% endif %}
-    {% endfor %}
+            {% if not values_only %}
```

Comment:
I was wondering if someone will ask for a `key_only` options.

Maybe it's better to chose an option `display` with the values: `key`, `value`, `both` ; with `both` as a default value. Or something like this.

Reply:
It makes sense, will fix.